src/endpoints/proposals-inbox/routes/proposals-inbox.route.ts

```diff
@@ -0,0 +1,81 @@
+import {
+	GetRoute,
+	Request,
+	Result,
+	RouteHandler,
+} from '@alien-worlds/api-core';
+
+import { config } from '../../../config';
+import { ProposalsInboxInput } from '../domain/models/proposals-inbox.input';
+import { ProposalsInboxOutput } from '../domain/models/proposals-inbox.output';
+import { ProposalsInboxRequestDto } from '../data/dtos/proposals-inbox.dto';
+
+/*imports*/
+
+/**
+ * @class
+ *
+ *
+ */
+export class ProposalsInboxRoute extends GetRoute {
+	public static create(handler: RouteHandler) {
+		return new ProposalsInboxRoute(handler);
+	}
+
+	private constructor(handler: RouteHandler) {
+		super('/v1/eosdac/proposals_inbox', handler, {
+			hooks: {
+				pre: parseRequestToControllerInput,
+				post: parseResultToControllerOutput,
+			},
+		});
+	}
+}
+
+/**
+ *
+ * @param {Request} request
+ * @returns
+ */
+export const parseRequestToControllerInput = (
+	request: Request<ProposalsInboxRequestDto>
+) => {
+	// parse DTO (query) to the options required by the controller method
+	return ProposalsInboxInput.fromRequest({
+		...request.query,
+		dac_id: config.eos.legacyDacs[0],
+	});
+};
+
+/**
+ *
+ * @param {Result<ProposalsInboxOutput>} result
+ * @returns
+ */
+export const parseResultToControllerOutput = (
+	result: Result<ProposalsInboxOutput>
+) => {
+	if (result.isFailure) {
+		const {
+			failure: { error },
+		} = result;
+		if (error instanceof Error) {
+			return {
+				status: 200,
+				body: ProposalsInboxOutput.create().toJson(),
+			};
+		}
+
+		return {
```

Comment:
Im not sure if you ever get to this point... cuz at line 62 you check if error is instance of Error which is by default always true. Maybe you wanted to check for some custom error type